### ISSUE TITLE
Travis build failure.

### ISSUE THREAD

**Issue by mitm001** [jMonkeyEngine/wiki#99]
```
$ export JAVA_HOME=~/oraclejdk8

$ export PATH="$JAVA_HOME/bin:$PATH"

$ ~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"

Ignoring license option: BCL -- using GPLv2+CE by default

install-jdk.sh 2019-07-17

Expected feature release number in range of 9 to 14, but got: 8

The command "~/bin/install-jdk.sh --target "/home/travis/oraclejdk8" --workspace "/home/travis/.cache/install-jdk" --feature "8" --license "BCL"" failed and exited with 3 during .

Your build has been stopped.
```

See this thread,
https://travis-ci.community/t/expected-feature-release-number-in-range-of-9-to-12-but-got-8-installing-oraclejdk8/1345

It is suggested to move to an oracle build in the provided range or if we want to stick to jdk8 use openjdk8.

Which way to go guys?

**Comment by Ali-RS** [comment]
@stephengold  what is your idea?

**Comment by davidB** [comment]
IMHO: try to switch to "openjdk11"

**Comment by stephengold** [comment]
I have no opinion on this.

**Comment by Ali-RS** [comment]
I made a PR (#100)  to switch to openjdk11 but the build fails. See https://travis-ci.org/jMonkeyEngine/wiki/builds/574499952

Do I need to update gradle version too? 


I have no experience with Travis build, will appreciate any help :)

**Comment by Ali-RS** [comment]
I made another PR (#101) to test openjdk8 also. Seems the build succeed. See https://travis-ci.org/jMonkeyEngine/wiki/builds/574506049

It's upto you guys which one we should select.

**Comment by mitm001** [comment]
Fixed now, good job...

**Comment by davidB** [comment]
IMHO (I no longer a jme community member) use openjdk8.
If you like, I'll do a PR (this week or next week to update openjdk, gradle,...) because the tool chain seems very similar to what I original provide ;-)

**Comment by mitm001** [comment]
He got it David.

https://github.com/jMonkeyEngine/wiki/pull/101

https://github.com/jMonkeyEngine/wiki/pull/100

**Comment by davidB** [comment]
+ maybe an integration with dependabot (I'm a fan of this service to keep project dependency uptodate)

**Comment by Ali-RS** [comment]
Okay, I upgraded to Gradle v5.5.1 and it fixed build with openjdk11.

Regarding dependabot, I have not used it before, but I am ok to use it if others have no objection.

**Comment by mitm001** [comment]
Kinda torn myself, first time in 3 years needed to change it and probably only because of oracles dumbass. Then again now things are on hyper-speed as far as updating goes...

**Comment by mitm001** [comment]
(I no longer a jme community member)

Thanks for still popping in and helping. Very glad you do.

**Comment by Ali-RS** [comment]
Yep, me too, very glad you are helping @davidB. 
Thank you :)

**Comment by davidB** [comment]
Thank you, but I did nothing except generating confusion ;-)
Happy to heard you guys.

**Comment by Ali-RS** [comment]
Fixed in #100
